packages/components/src/color-palette/test/index.tsx

```diff
@@ -156,15 +156,9 @@ describe( 'ColorPalette', () => {
 
 		expect( dropdownButton ).toBeVisible();
 
-		expect(
-			within( dropdownButton ).getByText( EXAMPLE_COLORS[ 0 ].name )
-		).toBeVisible();
+		expect( screen.getByText( EXAMPLE_COLORS[ 0 ].name ) ).toBeVisible();
```

Comment:
Originally it was checking to see if the text was rendered inside the button/swatch, basically making sure it's overlayed correctly. With the check moving outside of the button, I didn't think we'd need that check any longer.

Now that you mention it though, the test is specifically named `should render dropdown and its content`. The `dropdown` is the button/swatch... and it doesn't _have_ content any longer, because we're moving it.

I'll rename the test to be more in line with the changes of the PR... at the same time, I can add the `within` back. We'd just need to check that the name and value are `within` the new wrapper, and not dropdown. Do you think that would be helpful? Or is the `toBeVisible` check of the correct values sufficient?